contributing/branching-and-release-strategy.md

```diff
@@ -0,0 +1,79 @@
+# Branching and release strategy
+
+The Valtimo product aims to deliver features to its user quickly. Contributions from outside the core
+development team are welcomed. To make this possible, the below branching and release strategy is used.
+
+## Branching
+
+### `next-minor`
+
+Both [valtimo-frontend-libraries](https://github.com/valtimo-platform/valtimo-frontend-libraries) and [valtimo-backend-libraries](https://github.com/valtimo-platform/valtimo-backend-libraries/)
+are mainly developed in one single branch: `next-minor`. As its name implies, all commits made into `next-minor` will be
+released publicly when the next minor version of the product is released.
```

Comment:
```suggestion
included when the next minor version of the product is released.
```

Reply:
Maybe also mention that `next-minor` will also be merged into `next-major` when a minor is released?